### DISCUSSION TITLE
Default to panel instead of popout?

### DISCUSSION THREAD

**Discussion by olatheander** [ldelossa/litee.nvim#104] [Q&A] [answered]
Hi Louis,

Great work, just discovered the Litee suite of plug-ins. However, I'm a little confused on how to properly use it. I think I got the interaction with the `LTPanel` and e.g. `vim.lsp.buf.outgoing_calls()` but I cannot quite get the behavior I desire.

If I do a `lua vim.lsp.buf.outgoing_calls()` the popout shows with the calls and if I then do the `:LTPanel` the calls are docked in the panel but I haven't figured out how to go straight to panel and not have the popout since I cannot do the `LTPanel` (I then get the `Must open and Ltee component first...` message).

What I would desire is something like the nice right-hand panel with calls, explorer and bookmark on a simple keymap to toggle but not sure how to accomplish it.

**Comment by ldelossa** [comment, marked as answer]
Hey, 

Each specific plugin controls how their initial invocation opens. Check out the Calltree help file, the config parameter youre looking for is: https://github.com/ldelossa/litee-calltree.nvim/blob/25aa3be6a029519b68fa69d39d9ef104d8129455/doc/litee-calltree.txt#L198

**Reply by olatheander** [reply]
Super, thank you.
